src/common/interfaces.d.ts

```diff
@@ -0,0 +1,70 @@
+export interface LabeledProperties {
+	labelAfter?: boolean;
+	labelHidden?: boolean;
+	label?: string;
```

Comment:
Could this be a `DNode`? That would give a little more flexibility in usage (e.g. adding an icon after the text, or a red "*", etc)

Reply:
The icon / * would be achieved via the appropriate modifier classes on `label` itself. If the user wishes to use a more complicated `Label` type the can do this by manually creating and wrapping.
ie.

```ts
w(MySpecialLabel, {}, [
   w(TextInput, {})
])
```

Reply:
It's an easy non-breaking change to make in the future if we think it's necessary, so I'm willing to go with `string` for now and see if anyone complains